paper.tex

```diff
@@ -1929,8 +1983,43 @@ \section{Conclusion and Future Work}
 Kinkaku-ji. More extensive user testing at the actual site could have helped us
 catch bugs earlier and lead to a more polished final product.
 
-Despite these issues our team is happy with the product we delivered to
-Kyoto~VR. Selling ARuko on app stores was not within the scope of our academic
+While not a disaster, we had some initial trouble getting started with Unity.
+Different versions of Unity do not play well with each other, and it often runs
+into bugs on older hardware, such as the laptops we had to work with.
+Furthermore, a stable version of the Unity editor is not available on
+GNU/Linux~\autocite{best2019}, so one of our team members had to install macOS
+in order to use it. We were able to resolve these issues in the first weeks of
+the project, but it consumed some of our valuable time in July.
+
+\subsubsection{What Went Well}
+\label{sec:whatWentWell}
+
+Despite these issues, in the end we were able to create a useful working
+prototype. The Editour is fully featured with everything needed to build an
+audio tour anywhere in the world. ARuko allows users to experience useful
+AR-enhanced tours in a pleasant and intuitive way. Together they provide a
+complete platform for building augmented reality tour guide apps.
+
+Editour was not initially a requirement that Atticus had in mind, but it turned
+out to be a very useful tool in designing tours and allowing the app to be
+extensible. Two members of our team, Cole Granof and Joseph Petitti, had
+experience building web applications with HTML, CSS, and JavaScript, and the
+Editour allowed them to use these skills to help the project.
+
+Our team's broad range of skills made it so that each member had specific tasks
+that they were good at. This made the division of labor easier, and throughout
+the project we never had issues splitting up work.
```

Comment:
Maybe this paragraph should go before the previous one?

Reply:
Moved